diff --git a/va_explorer/va_data_management/views.py b/va_explorer/va_data_management/views.py
index 9a7cd13..8e84f1d 100644
--- a/va_explorer/va_data_management/views.py
+++ b/va_explorer/va_data_management/views.py
@@ -88,9 +88,6 @@ class Index(CustomAuthMixin, PermissionRequiredMixin, ListView):
             query = ', '.join([f"{k}: {query_dict[k]}" for k in query_keys if query_dict[k] != ""])
             write_va_log(LOGGER, f"[data_mgnt] Queried VAs for: {query}", self.request)
 
-
-
-
         return self.filterset.qs
 
     def get_context_data(self, **kwargs):
